### PR TITLE
Upgrade PuppetDB to 2.2.2

### PR DESCRIPTION
**Pull request by xaque208**

### CODE REVIEW

**Comment by koobs** [comment]
Looks good to me :+1: 

Have you run it through poudriere testport or bulk -t ?

**Comment by xaque208** [comment]
I have run it through testport and everything looked pretty clean.  I'm giving it another go now and reviewing.  It would be cool if we could build something that would auto test the ports tree and post the results of the build back here.

**Comment by xaque208** [comment]
All looks good to me using testport.

**Comment by koobs** [comment]
Oh, you're the maintainer of PuppetDB, nice :)

**Comment by xaque208** [comment]
I am, but I don't have commit on the tree nor merge here.  Is there something else I need to do to get this merged?

**Comment by koobs** [comment]
The standard process (at the moment) is to create an issue report @ Bugzilla. However, in this case, I'll take care of it from here and reference this PR on commit ;)

**Comment by xaque208** [comment]
Thank you.  Is the process documented anywhere?

**Comment by koobs** [comment]
[Porters Handbook](https://www.freebsd.org/doc/en_US.ISO8859-1/books/porters-handbook/book.html)

Specifically: [3.6. Submitting the New Port](https://www.freebsd.org/doc/en_US.ISO8859-1/books/porters-handbook/book.html#porting-submitting)

TLDR:

* svn diff against the latest version of the ports tree
* Create an issue report @ [Bugzilla](https://bugs.freebsd.org/bugzilla/) with the patch attached with title: **[MAINTAINER] category/port: Update to X.Y.Z** - Replace *category* and *port* appropriately
* Attach successful portlint and build (poudriere) output (note: attachments, not in comments)

**Comment by xaque208** [comment]
I guess I'm confused about where github plays a role.  I thought I'd just submit a PR and that would be the end of it.

**Comment by koobs** [comment]
We're working on closer integration between github -> bugzilla, so that newly created pull requests create new issues that can then be assigned, reviewed and committed by FreeBSD committers. Until that is complete, the repositories (at least src, doc and ports) under the Github FreeBSD organisation are read-only mirrors of the underlying Subversion repositories.

That basically means that pull-requests created here will be handled on a best-effort basis, for those FreeBSD Committers who keep an eye on what activity happens in Github :)

Having said that, it *is* one of our goals to make contribution via GitHub as easy as possible, so stay tuned

**Comment by xaque208** [comment]
@koobs That sounds good.  Is that workflow something that I can lend a hand on?  I was thinking some kind of automated testing would be useful here to perhaps send the results of a portlint or something to github PRs.  Just thoughts, and if its under control then no worries, but if that effort needs a hand, I'd like to lend it.

**Comment by mmoll** [comment]
I didn't see this, in the meanwhile I opened https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=198619
sorry :(

**Comment by xaque208** [comment]
So, did this get merged then?

**Comment by mmoll** [comment]
yes, this should get closed shortly. :)

**Comment by mmoll** [comment]
BTW, there's https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=199187 now and as I gained a FreeBSD ports commit bit recently, I intend to commit 2.3.3 in a few days (pending positive mentor review). As port maintainer you should get mails asking for maintainer-feedback, did anything arrive?

**Comment by xaque208** [comment]
I have received emails, but its not clear what I'm meant to do with them.  I've been waiting for this PR to get merged before I submitted the 2.3.3 update as well, so thats great if you have plans to do that already.

**Comment by mmoll** [comment]
I see. @xaque208 I'll ask for maintainer approval via mail, once I got everything up for 2.3.3, as I still have one question regarding PuppetDB anyway :)

**Comment by xaque208** [comment]
This has been merged out of band.
